app/components/common/userInput/index.tsx

```diff
@@ -6,8 +6,8 @@ interface UserInputProps {
 
 const UserInput: React.FC<UserInputProps> = ({ label, placeholder, link }) => (
   <main className="mb-6">
-    <p className="text-sm text-black mb-2">{label}</p>
```

Comment:
I believe this should not be removed without a discussion with @harshith-venkatesh as it is a common component and it is used at other places as well.

If you need to override `text-sm` without affecting other implementations, consider adding an additional prop `labelClassName` and pass it accordingly.

Reply:
Its a small css change and I looked it didn't affect anything major

Reply:
If that is the case, it would be better to keep the change as it is only